Service/SlidesInSlideDataCron.php

```diff
@@ -51,6 +51,14 @@ public function onCron(CronEvent $event)
     /** @var \Os2Display\CoreBundle\Entity\Slide $slide */
     foreach ($slidesOurType as $slide) {
 
+      /**
+       * We are running into issues, when we try to fetch data for more than ~20 slides concurrently. 
+       * This issued is cause by some security precautions from the external webserver. Therefore, 
+       * we added a tiny sleep delay between each call.
```

Comment:
```suggestion
       * We are running into issues, when we try to fetch data for more than ~25 slides in quick succession
       * during a cron run. This issued is specifically caused by the kk.dk multisite system which drops 
       * connections in such situations - probably to protect against DDoS attacks.
       * To avoid triggering this we added a tiny sleep delay between each call. It may not be the most 
       * efficient solution but it works for now.
```

I have tried to update the comment to make it a bit more precise.